### DISCUSSION TITLE
How to set Option variable to OFF ?

### DISCUSSION THREAD

**Discussion by ShelbyTang** [microsoft/vcpkg#27154] [Q&A]
when I want to install `curl` by `vcpkg` and set Option variable `ENABLE_UNICODE` 

<img width="856" alt="image" src="https://user-images.githubusercontent.com/54990505/194771432-299275ed-c20c-4daa-89a3-0b41c4fb04d6.png">

[link](https://github.com/curl/curl/blob/master/CMakeLists.txt#L85)

I try to set it on triplet but it doesn't work well, What should I do to set it?
```
set(VCPKG_TARGET_ARCHITECTURE x86)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE dynamic)

set(VCPKG_CXX_FLAGS "-D_CURL_TARGET_WINDOWS_VERSION=0x0501")
set(VCPKG_CXX_FLAGS "-D_ENABLE_INET_PTON=OFF")
set(VCPKG_C_FLAGS "-D_CURL_TARGET_WINDOWS_VERSION=0x0501")
set(VCPKG_C_FLAGS "-D_ENABLE_INET_PTON=OFF")
```

**Comment by Neumann-A** [comment]
```
if(PORT STREQUAL "curl")
  set(VCPKG_CMAKE_CONFIGURE_OPTIONS "-DENABLE_UNICODE=ON")
endif()
```

**Comment by dg0yt** [comment]
`-DENABLE_UNICODE=ON` is always set for windows, since #25865.
